.github/packaging/project/build_package.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+set -xeuo pipefail
+
+function build_packages(){
+  if [ "$ENV_DISTRO" = "" ]; then
+    echo "ENV_DISTRO is not set"
+    return
+  fi
+  chown -R root:root .
```

Comment:
This seems like an odd thing to do - what directory does this execute from and why is this needed?